change/@fluentui-react-charting-a3ef5de9-e169-4ae5-8380-461ba743a354.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add Viva dataviz gradient palette and rounded corners to charts",
+  "packageName": "@fluentui/react-charting",
+  "email": "t-shawngrant@microsoft.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
Multi stacked bar chart is left. 
https://developer.microsoft.com/en-us/fluentui#/controls/web/horizontalbarchart/multistackedbarchart

Reply:
I can see it now. Resolving this comment